CHANGES

```diff
@@ -18,6 +18,8 @@ Vernacular Commands
 
 - Removed deprecated commands Arguments Scope and Implicit Arguments
   (not the option). Use the Arguments command instead.
+- Nested proofs are not marked as deprecated anymore but using them
+  now requires option `Nested Proofs Allowed` to be turned on.
```

Comment:
Looks fine but maybe I would tweak the phrasing a bit more. "Using "nested proofs", that is to say, opening a `Lemma` inside the proof of another `Lemma` will produce now an error by default. You can keep using them by activating the `Nested Proof Allowed` option."

Reply:
Thanks for your comment. I'm unconvinced by my current formulation, but I'm not sure I really prefer the one you are proposing. Any other suggestion anyone?

Reply:
Nested proofs may be enabled through the option `...`. By default they are disabled and produce an error. The deprecation warning which used to occur when using nested proofs has been removed.

Reply:
Thanks @SkySkimmer!

Reply:
I think I prefer a phrasing that says first "now nested proofs will produce an error"

This is the most relevant bit of information for the user "action X, that used to work, now produces an error." Then, we should add some info how to fix it.